src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

```diff
@@ -58,10 +60,6 @@ public DescriptorImpl() {
             load();
         }
 
-        public boolean isApplicable(Class<? extends Job> jobType) {
-            return AbstractProject.class.isAssignableFrom(jobType);
```

Comment:
maybe point this to same scope as trigger? As we have plans to extend property with credentials, or maybe additional GH information. (In this case it should not break anything but will be strange in corner cases)